go/cloud-query/api/proto/cloudquery.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+
+// Determines the package name for the generated code.
+package cloudquery;
+
+// Configures where the generated code will be placed.
+option go_package = "internal/proto/cloudquery";
+
+message AwsCredentials {
+  string access_key_id = 1;
+  string secret_access_key = 2;
+}
+
+message AzureCredentials {
+  string subscription_id = 1;
+  string tenant_id = 2;
+  string client_id = 3;
+  string client_secret = 4;
+}
+
+message GcpCredentials {
+  string impersonation_token = 1;
```

Comment:
I think this should be a service account json file

Reply:
See: https://hub.steampipe.io/plugins/turbot/gcp

Reply:
We missed the info that it should be possible to use service account JSON without a file. They use the file path in most of the places but it looks like it should accept JSON as well.